.github/workflows/lint-test.yml

```diff
@@ -142,16 +142,28 @@ jobs:
 
     # This only runs if the previous steps were successful, no point in running it otherwise
     - name: Try building package
+      id: try-building-package
       run: |
         pdm install -dG build
 
         pdm run python -m build
         ls -la dist/
 
+    - name: Check built packages
+      if: steps.try-building-package.outcome == 'success'
+      run: |
         pdm run twine check --strict dist/*
 
+    - name: Upload built packages
+      if: always() && (steps.try-building-package.outcome == 'success' || steps.try-building-package.outcome == 'failure')
+      uses: actions/upload-artifact@v4
+      with:
+        name: disnake-wheel-${{ hashFiles('disnake/**.py') }}
```

Comment:
Is there a reason to use hashFiles in particular, or would something like the run ID or commit hash also work?

Reply:
Hm, the run ID or the commit hash would also work, but where's the fun in that?